src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayloadV3.cs

```diff
@@ -36,6 +37,7 @@ public override bool TryGetBlock([NotNullWhen(true)] out Block? block, UInt256?
         block.Header.ParentBeaconBlockRoot = ParentBeaconBlockRoot;
         block.Header.BlobGasUsed = BlobGasUsed;
         block.Header.ExcessBlobGas = ExcessBlobGas;
+        block.Header.RequestsHash = ExecutionRequests != null ? ExecutionRequests.CalculateRoot() : Hash256.Zero;
```

Comment:
do you know why it was decided in this way? (No V4, just V3?)

Reply:
https://github.com/ethereum/execution-apis/pull/591/files#diff-18471a795328cf06e000056d594a646c71fa9ce16c35511f3506543b6fc57981

I think we are not going with the new version of `ExecutionPayload` because we are exchanging `Requests` as an additional parameter. With the removal of all kinds of requests from the proposed `ExecutionPayloadV4`, it becomes essentially `ExecutioPayloadV3`.